network-api/networkapi/mozfest/templates/partials/streamfield.html

```diff
@@ -1,5 +1,9 @@
 {% load wagtailcore_tags %}
 
 {% for block in page.body %}
+    {# optionally render a spacer if the first block is a paragraph. Only for landing_page / homepage page types with the dark hero #}
+    {% if forloop.first and block.block_type == 'paragraph' %}
+        {% block body_spacer %}{% endblock body_spacer %}
+    {% endif %}
```

Comment:
Hey @robdivincenzo, I think the paragraph block needs a spacer every time, not only when it's the first block:

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/61277874/98363f96-9a71-497e-b19d-91203b7b3b3f)

Reply:
Thanks for reviewing @jhonatan-lopes ! I've made a couple commits with your feedback to include the paragraph spacer each time on these dark themed pages, and then also to include a post_paragraph spacer so that spacing is balanced.

![Screenshot 2024-02-16 at 8 59 37 AM](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/3083419/d4ec2390-7bca-431f-a6b8-05ca983ed2c2)